src/lib/ITensorMPS/src/mpo.jl

```diff
@@ -662,11 +662,16 @@ Choose the method with the `method` keyword, for example
 - `mindim::Int=1`: the minimal bond dimension of the resulting MPS.
 - `normalize::Bool=false`: whether or not to normalize the resulting MPS.
 - `method::String="densitymatrix"`: the algorithm to use for the contraction.
-   Currently the options are "densitymatrix", where the network formed by the
-   MPO and MPS is squared and contracted down to a density matrix which is
-   diagonalized iteratively at each site, and "naive", where the MPO and MPS
-   tensor are contracted exactly at each site and then a truncation of the
-   resulting MPS is performed.
+    - "densitymatrix": The network formed by the MPO and MPS is squared and contracted down to
+    a density matrix which is diagonalized iteratively at each site.
+    - "naive": The MPO and MPS tensor are contracted exactly at each site and then a truncation 
+    of the resulting MPS is performed.
+    - "zipup": The MPO and MPS tensors are contracted then truncated at each site without enforcing
+    the appropriate orthogonal gauge. Once this sweep is complete a call to `truncate!` occurs.
+    Because the initial truncation is not locally optimal it is recommended to use a loose
+    `cutoff` and `maxdim` and then pass the desired truncation parameters to the locally optimal
+    `truncate!` sweep via the additional keyword argument `truncate_kwargs`.
+    Suggested use is `contract(A, ψ; method="zipup", cutoff=cutoff / 10, maxdim=2 * maxdim, truncate_kwargs=(; cutoff, maxdim))`.
```

Comment:
```suggestion
    For example, `contract(A, ψ; method="zipup", cutoff=cutoff / 10, maxdim=2 * maxdim, truncate_kwargs=(; cutoff, maxdim))`.
```
I think "suggested" is a bit strong, users often take suggestions in docs very seriously and don't play around with things themselves, and doubling the bond dimension could lead to very large bond dimensions in some cases.

Reply:
This was scheme was suggested in [Time-evolution methods for matrix-product states](https://doi.org/10.1016/j.aop.2019.167998) just below equation 39

>
> For the relaxed truncation scheme a maximal growth factor
of the MPS bond dimension m′ = 2m and a truncated weight of 1/10 of the target weight turns out
to be a suitable choice. The contraction then has complexity 2m3wσ + 2m2w2σ 2. The main cost is
in the SVD of a 2mσ × mw matrix at cost O(m3σ w), i.e. linear in w. The MPS is in left canonical
form now. A subsequent compression as described in Section 2.6 should be applied to obtain the
resulting MPS with the target bond dimension.

So I do think there is some value to these specific parameters beyond simply demonstrating how to pass different truncation arguments. But of course I will defer to your preference.

Reply:
In that case, we could cite that paper and say: "A set of parameters suggested in [1] is ...".

Reply:
I like that.

Reply:
Done.